package.json

```diff
@@ -63,20 +66,34 @@
   },
   "config": {
     "commitizen": {
-      "path": "cz-conventional-changelog"
+      "path": "node_modules/cz-conventional-changelog"
     }
   },
   "commitlint": {
     "extends": [
       "@commitlint/config-conventional"
-    ]
+    ],
+    "rules": {
+      "footer-max-line-length": [
```

Comment:
it throws an error, please refer to this https://github.com/e-Spirit/crownpeak-pwa-template/actions/runs/5860564569/job/15888907727. it is a workaround  how to avoid that error